### DISCUSSION TITLE
Expired domain renewal

### DISCUSSION THREAD

**Discussion by jpriceoenww** [vercel/community#6517] [Help]
### Summary

I have a domain registered via Vercel (winternet.work) that failed to renew due to an out of date payment method.  I have updated that payment method but I do not see any way on my team's dashboard to renew the domain now that it has expired.  

Any help with this is greatly appreciated

### Steps to Reproduce

Log in, go to domains, see expired domain, do not see option to renew during grace period.

**Comment by amyegan** [comment]
Hey @jpriceoenww. Please [submit a **Domain Renewals** case](https://vercel.com/help) to the Customer Success Team. That will get open a ticket with our billing experts who can help with this 🙏

**Reply by jpriceoenww** [reply]
Thank you for the info.  I've already done this but I have had no had a response as of yet nor have I gotten an email confirmation that my ticket was created.  I did finally figure out how to view open tickets after I had opened a 2nd ticket (which someone has now merged with the first ticket).
